docs/source/authn-guide/custom-ldap-replication.md

```diff
@@ -0,0 +1,82 @@
+# Authenticating via an External IDP a New User Created Using a Custom Authentication Script in an Opendj Replication Setup
+
+## General Overview
+
+Using [Interception scripts](https://gluu.org/docs/gluu-server/4.3/authn-guide/customauthn/), you can customize many aspects of your Gluu Server identity and access management services for example using a custom script to create and authenticate a user.
+
+This section explains how to authenticate via an external IDP a new user that has been created with a custom interception script in a multi cluster setup of Gluu using LDAP as the backend persistence. For reference, you can read how a GLUU LDAP cloud native multi cluster setup is implemented [here](https://gluu.org/docs/gluu-server/4.3/tutorials/cn-ldap-multi-cluster/#test-replication). 
```

Comment:
not a mutli cluster setup. This is a normal cloud native (kubernetes) setup, so point to the kubernetes installation docs.